pkg/integration/components/matcher.go

```diff
@@ -61,6 +61,11 @@ func (self *matcher) Contains(target string) *matcher {
 	return self.appendRule(matcherRule{
 		name: fmt.Sprintf("contains '%s'", target),
 		testFn: func(value string) (bool, string) {
+			// everything contains the empty string so we unconditionally return true here
+			if target == "" {
```

Comment:
no idea why this code was needed: I thought strings.Contains would do the job properly but apparently not